iot-hub/Samples/service/EdgeDeploymentSample/EdgeDeploymentSample.cs

```diff
@@ -102,12 +102,9 @@ private async Task CleanUpConfigurations()
                 foreach (var configuration in configurations)
                 {
                     string configurationId = configuration.Id;
-                    foreach (var prefix in ConfigurationIdPrefix)
```

Comment:
@abhipsaMisra When I was going over this sample, I believe this foreach was added by mistake. This line simply loops over the characters in ConfigurationIdPrefix. We are already looping over the configurations on line 102.